src/components/CallToActionsBar.tsx

```diff
@@ -13,9 +13,10 @@ export interface CallToAction {
 interface Props {
   callToActions: CallToAction[]
   testID?: string
+  index?: number
```

Comment:
what do you think of `positionInNotificationList` instead of `index` so that it's easier to understand what the index is relative to?